app/views/artifacts/index.html.erb

```diff
@@ -4,8 +4,9 @@
   <h3><%= artifact.name %></h3> 
   <p>Material: <%= artifact.material %></p>
   <p>Date Created: <%= artifact.year_created %></p>
-  <p>Total pieces: <%= artifact.total_pieces %></p>
+  <p>Total Pieces: <%= artifact.total_pieces %></p>
```

Comment:
I have changed this multiple times, not sure why it continues to revert back to old code??